common/flatpak-utils-base.c

```diff
@@ -28,6 +28,18 @@
 #include <gio/gio.h>
 #include "libglnx.h"
 
+char *
+flatpak_get_tzdir (void)
+{
+  const gchar *tzdir;
+
+  tzdir = getenv ("TZDIR");
+  if (tzdir)
+    return tzdir;
+
+  return "/usr/share/zoneinfo";
```

Comment:
I think you can simplify this with the Elvis operator:

```c
  const gchar *tzdir = getenv ("TZDIR");
  return tzdir ?: "/usr/share/zoneinfo";
```

Reply:
I've never in my live seen this and I'm not sure if I want to give others a big wtf moment.

Reply:
It's a GCC (then Clang) thing.